auditcare/management/commands/compute_diffs.py

```diff
@@ -1,15 +1,20 @@
-from optparse import make_option
 from django.core.management.base import  NoArgsCommand
 from auditcare.models import AccessAudit, ModelActionAudit
 
 class Command(NoArgsCommand):
```

Comment:
https://docs.djangoproject.com/en/1.10/releases/1.10/#features-removed-in-1-10

The class django.core.management.NoArgsCommand is removed.